shared/productResponse.ts

```diff
@@ -119,6 +119,20 @@ export function isProduct(
 	return productTiers.includes((data as ProductDetail)?.tier);
 }
 
+export const isGuardianProduct = (productDetail: ProductDetail): boolean => {
+	return (
+		productDetail.tier !== 'Newspaper Delivery - Observer' &&
+		productDetail.tier !== 'Newspaper Digital Voucher - Observer'
+	);
+};
```

Comment:
can we just reuse rather than reimplementing?
```suggestion
export const isGuardianProduct = (productDetail: ProductDetail): boolean => !isObserverProduct(productDetail);
```

Reply:
Your suggestion to use the array.every method negates the need for the `isGuardianProduct` entirely so I've removed it 👍